tests/test_potodo.py

```diff
@@ -47,7 +42,7 @@ def test_json_output(capsys):
             "files": [
                 {
                     "name": "folder/file3",
-                    "path": str(FIXTURE_DIR / REPO_DIR / "folder" / "file3.po"),
+                    "path": f"{ABS_REPO_DIR}/folder/file3.po",
```

Comment:
Je ne peux pas tester, je n'ai que des Linux, mais j'imagine que sur d'autres OS :

    f"{ABS_REPO_DIR}/folder/file3.po" != f"{ABS_REPO_DIR / 'folder' / 'file3.po'}"

J'aime beaucoup les f-strings, mais je trouve ça toujours un peu exagéré de les utiliser pour remplacer str, je veux dire 

    str(x) == f"{x}"

est vrai, mais je préfère le premier, donc si tu le corrige, je conseillerai :

```suggestion
                    "path": str(ABS_REPO_DIR / "folder" / "file3.po"),
```

Le risque est faible, mais j'imagine que les tests échouent sur Windows sans ce patch, et passent avec, peut etre.

Reply:
À moins d'être sur une version de Windows antérieure à XP, il me semble que les « / » sont [autorisés](https://docs.microsoft.com/en-us/dotnet/standard/io/file-path-formats#canonicalizing-separators).

Reply:
Peut être :)

Reply:
Mais comme tu l'as dit « osef c'est Windows ».